doc/source/guide/derivatives_guide.rst

```diff
@@ -52,25 +57,31 @@ Here :math:`f'(x)` is defined as the linear operator (if it exists) that satisfi
 
 This definition of the derivative is called the *Fréchet derivative*.
 
+The Gateaux derivative
+~~~~~~~~~~~~~~~~~~~~~~
+The concept of directional derivative can also be extended to Banach spaces, giving the *Gateaux* derivative. 
+The Gateaux derivative is more general than the Fréchet derivative, but is not a linear operator. An example of a function that is Gateaux differentiable but not Fréchet differentiable is the absolute value function.
+It not used in ODL since it can fail to be linear. 
```

Comment:
Will fix the formatting to what we currently do (i.e. actually allow Gateux derivative, but prefer Frechet).